components/script/dom/htmlimageelement.rs

```diff
@@ -673,23 +673,25 @@ impl VirtualMethods for HTMLImageElement {
                return;
            };
 
-           let point = Point2D::new(mouse_event.ClientX().to_f32().unwrap(), mouse_event.ClientY().to_f32().unwrap());
+           let point = Point2D::new(mouse_event.ClientX().to_f32().unwrap(),
+                                    mouse_event.ClientY().to_f32().unwrap());
+
            // Walk HTMLAreaElements
-           let mut index = 0;
-           while index < elements.len() {
-               let shape = elements[index].get_shape_from_coords();
+           for element in elements {
+               let shape = element.get_shape_from_coords();
                let p = Point2D::new(self.upcast::<Element>().GetBoundingClientRect().X() as f32,
-               self.upcast::<Element>().GetBoundingClientRect().Y() as f32);
+                                    self.upcast::<Element>().GetBoundingClientRect().Y() as f32);
+
                let shp = if let Some(x) = shape {
                    x.absolute_coords(p)
                } else {
                    return
                };
+               
```

Comment:
I actually saw Intellij had a little marker in the line number bar on that line but didn't know what it meant, TIL.